nixos/modules/security/acme/default.nix

```
@@ -990,12 +982,9 @@ in {
           then roundRobinApplyAttrs selfsignServiceFunctions concurrencyLockfiles
           else lib.mapAttrs (_: f: f null) selfsignServiceFunctions;
         in
-        { "acme-fixperms" = userMigrationService; }
-        // (lib.optionalAttrs (cfg.maxConcurrentRenewals > 0) {"acme-lockfiles" = lockfilePrepareService; })
+        { acme-setup = setupService; }
         // renewServices
-        // (lib.optionalAttrs (cfg.preliminarySelfsigned) ({
-        "acme-selfsigned-ca" = selfsignCAService;
-      } // selfsignServices));
+        // (lib.optionalAttrs cfg.preliminarySelfsigned selfsignServices);
```

Style nit:
```suggestion
        // lib.optionalAttrs cfg.preliminarySelfsigned selfsignServices;
```
(Noticed while looking at the tests PR)

---

Those brackets are needed, no? Otherwise it's going to try wrapping `lib.optionalAttrs`'s definition instead of the result?

---

Not sure what I meant either :sweat_smile:  I think I meant to say evaluating. Regardless - thanks for the explanation. Will change it.